test/physics/em/PhotoelectricInteractor.test.cc

```diff
@@ -0,0 +1,396 @@
+            unsigned int num_shells;
+            double       threshold;
+            infile >> num_shells >> num_shells >> threshold;
+            result.thresh_low = MevEnergy{threshold};
+            result.shells.resize(num_shells);
+
+            // Read the binding energies and fit parameters
+            for (auto& shell : result.shells)
+            {
+                double binding_energy;
+                infile >> binding_energy;
+                shell.binding_energy = MevEnergy{binding_energy};
+                shell.param_low.resize(6);
+                for (auto i : celeritas::range(6))
+                {
+                    infile >> shell.param_low[i];
+                }
+            }
+            infile.close();
+        }
+
+        // Read subshell cross sections fit parameters in high energy interval
+        {
+            std::ostringstream os;
+            os << "pe-high-" << atomic_number << ".dat";
+            std::string filename
+                = this->test_data_path("physics/em", os.str().c_str());
+            std::ifstream infile(filename.c_str());
+            CHECK(infile.is_open());
+
+            // Read the number of subshells and energy threshold
+            unsigned int num_shells;
+            double       threshold;
+            infile >> num_shells >> num_shells >> threshold;
+            result.thresh_high = MevEnergy{threshold};
+            CHECK(num_shells == result.shells.size());
+
+            // Read the binding energies and fit parameters
+            for (auto& shell : result.shells)
+            {
+                double binding_energy;
+                infile >> binding_energy;
+                CHECK(binding_energy == shell.binding_energy.value());
+                shell.param_high.resize(6);
+                for (auto i : celeritas::range(6))
+                {
+                    infile >> shell.param_high[i];
+                }
+            }
+            infile.close();
+        }
+
+        // Read tabulated subshell cross sections
+        {
+            std::ostringstream os;
+            os << "pe-ss-cs-" << atomic_number << ".dat";
+            std::string filename
+                = this->test_data_path("physics/em", os.str().c_str());
+            std::ifstream infile(filename.c_str());
+            CHECK(infile.is_open());
+
+            // Read tabulated subshell cross sections
+            for (auto& shell : result.shells)
+            {
+                double       min_energy, max_energy;
+                unsigned int size, shell_id;
+                infile >> min_energy >> max_energy >> size >> shell_id;
+                shell.energy.resize(size);
+                shell.xs.resize(size);
+                for (auto i : celeritas::range(size))
+                {
+                    infile >> shell.energy[i] >> shell.xs[i];
+                    shell.xs[i] *= barn;
+                }
+            }
+            infile.close();
+        }
```

Comment:
Is this the actual data format that Geant4 uses or a temporary format? If the former, per our discussion last week, we should probably just move this to a standalone class in `src/io` for use by the geant exporter.

Reply:
Yes, these are the actual data files. Sounds good, I'll add a separate class for reading the data.